test/test_pub.dart

```diff
@@ -312,7 +312,7 @@ Future runPub(
       args: args, workingDirectory: workingDirectory, environment: environment);
   await pub.shouldExit(exitCode);
 
-  expect(() async {
+  await expectLater(() async {
```

Comment:
FYI As long as the future returned from `runPub` is awaited in the tests the `await expectLater(..., completes);` isn't doing anything worthwhile.

Reply:
👍